tox.ini

```diff
@@ -4,17 +4,15 @@
 # When updating the envlist, be sure to also update TOX_ENV in .travis.yml
 envlist =
     docs,
-    {py27,py34}-django{1.6,1.7,1.8}-drf{3.0,3.1,3.2}
+    {py27,py34,py35}-django{1.8,1.9}-drf{3.2,3.3}
 
 [testenv]
 commands = {envpython} runtests.py []
 deps =
-    django1.6: Django==1.6.3
-    django1.7: Django==1.7.10
-    django1.8: Django==1.8.4
-    drf3.0: djangorestframework==3.0.5
-    drf3.1: djangorestframework==3.1.3
-    drf3.2: djangorestframework==3.2.2
+    django1.8: Django==1.8.8
```

Comment:
while you're in here, could you fix all these requirement strings so that they get bug fix releases? For example: `Django >= 1.8, < 1.9`

I don't know why they weren't already, probably just an oversight.